tf_encrypted/keras/layers/convolutional.py

```diff
@@ -157,12 +157,14 @@ def build(self, input_shape):
     self.kernel_shape = self.kernel_size + (input_dim, self.filters)
 
     kernel = self.kernel_initializer(self.kernel_shape)
-    self.kernel = self.prot.define_private_variable(kernel)
+    self.kernel = self.add_weight(
```

Comment:
I'd expect any call to `define_private_variable` to happen in `add_weight` instead